Caboodle/Geolocation/Geolocation.ios.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using CoreLocation;
+using Foundation;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Geolocation
+    {
+        internal static bool IsSupported
+            => CLLocationManager.LocationServicesEnabled;
+
+        static async Task<Location> PlatformLastKnownLocationAsync()
+        {
+            await Permissions.RequireAsync(PermissionType.LocationWhenInUse).ConfigureAwait(false);
+
+            var manager = new CLLocationManager();
+            var location = manager.Location;
+
+            return new Location
```

Comment:
Can you make this an extension method to convert like i do for geocoding. It may already be there actually.